docker-compose.yml

```diff
@@ -21,6 +21,30 @@ services:
             - ./api-dummy/public:/srv/app/public
         depends_on:
             - api-dummy-fpm
+    article-store:
+        build:
+            context: ./article_store
+        command: >
+                /bin/bash -c "
+                    until echo > /dev/tcp/postgres/5432; do sleep 1; done
+                    pipenv run python manage.py migrate articles
```

Comment:
why two separate `migrate` commands? As a Django newbie here

Reply:
It's actually an artefact of initial work, its no longer required. Will reduce to just `migrate`.